src/init.js

```diff
@@ -23,10 +23,14 @@ import { config } from "@onflow/config";
  * Inits framework variables, storing private key of service account and base path
  * where Cadence files are stored.
  * @param {string} basePath - path to the folder with Cadence files to be tested.
- * @param {number} port - port to use for accessAPI
+ * @param {number} [props.port] - port to use for accessAPI
+ * @param {number} [props.pkey] - private key to use for service account in case of collisions
  */
-export const init = async (basePath, port = 8080) => {
-  set("PRIVATE_KEY", process.env.PK, "accounts/emulator-account/keys");
+export const init = async (basePath, props = {}) => {
+  const { port = 8080 } = props;
+  const { pkey = "48a1f554aeebf6bf9fe0d7b5b79d080700b073ee77909973ea0b2f6fbc902" } = props;
```

Comment:
Is this meant to be hardcoded?

Reply:
We need some value as fallback, so emulator would start up 🤔